package.json

```diff
@@ -24,7 +24,7 @@
     "@solana/web3.js": "^1.20.2",
     "@types/redux-persist": "^4.3.1",
     "@vitejs/plugin-react-refresh": "^1.3.4",
-    "animated-number-react": "0.1.1",
+    "animated-number-react": "^0.1.1",
```

Comment:
version 0.1.1 should stay, 0.1.2 has a bug because of which every time animations start from 0